gossip/hub/handlers.go

```diff
@@ -96,6 +104,32 @@ func ctSTHChecker(data, sig []byte) error {
 	return nil
 }
 
+// ctSTHUpdateLatest indicates whether the 'latest' entry should be updated based on
+// the tree size and timestamp held in the CT Log STH.
+func ctSTHUpdateLatest(prev *api.TimestampedEntry, current *api.TimestampedEntry) bool {
```

Comment:
This looks like a "GreaterThan" function in the style of the [sort](https://golang.org/pkg/sort/#SliceIsSorted) package.  Would it be clearer to name it `Greater` ?

Reply:
Yeah, those functions have fallen into the trap of describing what other code uses them for, not what they actually do.  Renamed the function pointer to `entryIsLater` with implementations `rawEntryIsLater` and `ctSTHIsLater`.